test/unit/app/models/publishes_to_publishing_api_test.rb

```diff
@@ -101,6 +101,16 @@ def test_published_gone_handler; end
     test_object.publish_gone_to_publishing_api
   end
 
+  test "enqueues a worker to bulk republish the document on request" do
+    test_object = TestObject.new
+    class << test_object
+      include PublishesToPublishingApi
+    end
+
+    Whitehall::PublishingApi.expects(:bulk_republish_async).with(test_object).once
+    test_object.bulk_republish_to_publishing_api_async
+  end
```

Comment:
Should we have a context/test here covering `can_publish_to_publishing_api? => false`?

Reply:
Good idea.  I've added tests for both the republishing methods.